app/dashboard/src/pages/dashboard/Settings/MembersTable.tsx

```diff
@@ -10,7 +10,7 @@ import * as toastAndLogHooks from '#/hooks/toastAndLogHooks'
 import * as authProvider from '#/providers/AuthProvider'
 import * as textProvider from '#/providers/TextProvider'
 
-import UserRow from '#/layouts/Settings/UserRow'
+import UserRow from '#/pages/dashboard/Settings/UserRow'
```

Comment:
Can we please use relative imports for internal components? 

What an internal component is:

we have a SettingsPage. Everything under this page uses relative imports when imports siblings/nested components. If we import outside things, we use absolute imports.

Reply:
makes sense, i've moved `Settings.tsx` -> `Settings/index.tsx` and `Drive.tsx` -> `Drive/index.tsx` as well